package.json

```diff
@@ -2,15 +2,14 @@
   "private": true,
   "scripts": {
     "clean": "rimraf output && rimraf .pulp-cache",
-    "build": "jshint src && jscs src && pulp build --censor-lib --strict",
-    "test": "pulp build -I test"
```

Comment:
Was removing the `test` script intentional? Looks like the code for the tests was updated in this PR